legal-api/migrations/versions/5238dd8fb805_.py

```diff
@@ -62,11 +62,63 @@ def upgrade():
 
     # enum added after creating table as DuplicateObject error would be thrown otherwise
     op.add_column('amalgamating_businesses', sa.Column('role', role_enum, nullable=False))
-
+    
+    op.create_table(
+        'amalgamations_version',
+        sa.Column('id', sa.Integer(), nullable=False),
```

Comment:
You will need to following the new versioning model.

Take a look at [entity_roles](https://github.com/bcgov/lear/blob/93d6b09e56b7d8cec80e4e158abcd997e33e9f1d/legal-api/migrations/versions/004f7558063f_second.py#L685-L727) as an example.

So a couple of things to make note of:
- versioning tables are suffixed with `_history` as opposed to `_version` now.
- `transaction_id`,  `end_transaction_id` and `operating_type` are no longer used
- keeping the `filing_id` should be sufficient to trace the changes in place of `transaction_id` in the new versioning  approach